src/main/java/peoplesoft/ui/scenes/HelpPage.java

```diff
@@ -132,6 +84,8 @@ private void openInBrowser() {
             logger.warning("The URL in the application was wrong. Please contact developers.");
         } catch (IOException ie) {
             logger.warning("Unable to launch OS's browser. Please contact developers.");
```

Comment:
actually will the users see this message? or will they just see nothing happening (?) despite clicking the button multiple times

Reply:
Fixed this to allow that to happen.